week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,18 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
-}
-
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+};
 
+function introduceMe(name, age) {
+return `Hello, my name is ${name} and I am ${age} years old`;
+};
 function getRemainder(a, b) {
-    remainder = a %% b;
+    remainder = a % b;
```

Comment:
Good! Always declare your variables as `const` or `let`. It is good practice to do this and you will soon understand the value of it once we get into more advanced topics.